c3d/c3d.py

```diff
@@ -1434,14 +1629,14 @@ def seek_param_section_header():
                 desc = size and buf.read(size) or ''
                 group = self.get(group_id)
                 if group is not None:
-                    self.rename_group(group, name)
+                    self.rename_group(group, name)  # Inserts name key
                     group.desc = desc
                 else:
                     self.add_group(group_id, name, desc)
 
         self._check_metadata()
 
-    def read_frames(self, copy=True):
+    def read_frames(self, copy=True, analog_transform=True, camera_sum=False, check_nan=True):
```

Comment:
The newe parameters are not documented in the doc-string

Reply:
Copied docstring from future commit, class names are named based on future package structure however.